yarn-project/prover-node/src/factory.ts

```diff
@@ -40,8 +40,8 @@ export async function createProverNode(
   const publisher = new L1Publisher(config, telemetry);
 
   const txProvider = deps.aztecNodeTxProvider
-    ? new AztecNodeTxProvider(deps.aztecNodeTxProvider)
-    : createTxProvider(config);
+    ? new AztecNodeProverCoordination(deps.aztecNodeTxProvider)
+    : createProverCoordination(config);
```

Comment:
Nit: maybe rename `ProverCoordination` to `ProofCoordinator` or `ProverSelector`?